hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_matcher.py

```diff
@@ -103,58 +107,62 @@ def lambda_handler(event, context):
                 )
                 privacy_group_list = metadata.get("privacy_groups", [])
                 metadata["privacy_groups"] = filter(
```

Comment:
BTW, just found out. The filter() does not produce an actual list. You need to do `list(filter(x))` to get a list.